tbcrawler/pytbcrawler.py

```diff
@@ -55,6 +57,11 @@ def run():
     job_config = ut.get_dict_subconfig(config, args.config, "job")
     job = crawler_mod.CrawlJob(job_config, url_list)
 
+    # Run display
+    xvfb_wrapper = XvfbWrapper()
+    xvfb_wrapper._from(args.virtual_display)
+    xvfb_wrapper.start()
+
```

Comment:
I like the class because now I can close the correct display without caring about managing the returning variable of start_xvfb. I extracted a class because there is state, but I don't mind changing it back.

There is input checking in the _from: do you mean you'd rather like to put it in the caller, in pytbcrawler?